src/interactive/README.md

```diff
@@ -104,3 +113,79 @@ Only clears the appended suggestion when you restart the interactive shell
 ```
 
 The color option will be saved.
+
+# Intelligent AZ Interactive (Revolutionary Change)
+## Recommendation
+
+We have integrated the cli recommendation to make the completion ability more intelligent and provide the scenario completion. This is a revolutionary change. Users can enable or disable the recommendation feature by running the following commands:
+
+```
+$ az config set interactive.enable_recommender=True # Default, try the new recommendation feature
+$ az config set interactive.enable_recommender=False # Disable the recommendation feature
+```
+
+### Command Recommendation
+
+Command recommendation uses an intelligent algorithm to suggest the most relevant and frequently used command for the user based on their historical command inputs. By analyzing the user's input history and matching it with the highest relevant commands, Intelligent Az Interactive can assist Azure CLI beginners in making quick and informed decisions, saving them time and effort.
+
+The command recommendation feature can also be a good way to broaden users' knowledge boundaries and help them recognize and learn commands that are related to their common commands but never used before.
+
+### Scenario Identification
+
+Scenario identification is another powerful feature that automatically recognizes the user's current scenario and recommends a set of commands that are most relevant to the user's needs. This feature is based on our extensive database of over 600+ business scenarios, which cover a wide range of use cases across various scenes. By analyzing the user's input history, Intelligent Az Interactive can identify the relevant scenario and suggest the most appropriate command set, helping new CLI users to quickly accomplish their tasks with confidence and ease.
+
+## Search for Scenarios
+
+We have added the ability to help users search for scenarios based on keywords. When the user enters some keywords or descriptions, we will recommend the scenarios that are most likely to be used based on the keywords and descriptions of the functions the user wants to implement.
```

Comment:
We need to inform customers in the `README.md` how to use or trigger these new features

Reply:
Thank you for the suggestion, already modified Please refer to: https://github.com/wu5bocheng/azure-cli-extensions/blob/interactive-main/src/interactive/README.md